nucliadb_performance/nucliadb_performance/test_search_cluster.py

```diff
@@ -80,9 +109,37 @@ async def test_find(session):
 @scenario(weight=SEARCH_WEIGHT)
 async def test_search(session):
     client = get_search_client(session)
-    kbid = get_random_kb()
-    await client.make_request(
+    kbid = get_kb(session)
+    await make_kbid_request(
+        client,
+        kbid,
         "GET",
         f"/v1/kb/{kbid}/search",
         params={"query": fake.sentence()},
     )
+
+
+@global_teardown()
+def end_test():
+    print("This is the end of the test.")
+    for error in ERRORS:
+        print(error)
+
+
+async def make_kbid_request(client, kbid, method, path, params=None, json=None):
+    global ERRORS
+    try:
+        await client.make_request(method, path, params=params, json=json)
+    except RequestError as err:
+        # Store error info so we can inspect after the script runs
+        detail = (
+            err.content and err.content.get("detail", None) if err.content else err.text
+        )
+        error = Error(
+            kbid=kbid,
+            endpoint=path.split("/")[-1],
+            status_code=err.status,
+            error=detail,
+        )
+        ERRORS.append(error)
```

Comment:
That makes me think I could provide functions to print in the console (the right side is the errors) so you can also have them while running